woocommerce/woocommerce-ios
# [Mobile Payments] Add launch argument to enable verbose logging for Stripe

StripeTerminal offers some the option to enable verbose logging, which can be helpful when trying to debug issues related to card reader connection.

Instead of modifying the code every time we want to debug, this introduces a new launch argument that's easier to toggle in the Xcode scheme:

![Screen Shot 2021-06-03 at 14 09 16](https://user-images.githubusercontent.com/8739/120642837-b1509c80-c475-11eb-968d-d4fb68dc1d2f.png)

## To test

Start connecting to a card reader.

- With the argument disabled, no 💳 logs should appear in the console
- With the argument enabled, you should see all the extra 💳 logging

![Screen Shot 2021-06-03 at 14 22 17](https://user-images.githubusercontent.com/8739/120644002-1eb0fd00-c477-11eb-82b0-e2a5bac29aa2.png)


Update release notes:
